cabal-testsuite/PackageTests/PathsModule/ImportQualifiedPost/my.cabal

```diff
@@ -0,0 +1,18 @@
+name: PathsModule
+version: 0.1
+license: BSD3
+author: Martijn Bastiaan
+stability: stable
```

Comment:
`stability` is certainly superfluous, and folks are disputing it.

Reply:
Heh, yeah I just copied it from another cabal file in the testsuite